src/routes/feedback/components/FeedbackModal.js

```diff
@@ -53,7 +53,7 @@ type PropsType = {
   id?: number,
   alias?: string,
   query?: string,
-  feedbackStatus: string,
+  feedbackStatus: ?string,
```

Comment:
This only allows for nulls, still not for undefined...

Reply:
No, see https://flow.org/en/docs/types/maybe. Maybe types accept the provided type as well as null or undefined. So ?number would mean number, null, or undefined.